src/integration_tests/integration_tests.py

```diff
@@ -54,6 +55,38 @@ def test_dataset(client, bucket_loader):
     }
 
 
+def test_dataset_metadata(client, bucket_loader):
```

Comment:
So I would prefer you integrate with the existing dataset test. I've already done some work to get the sleep period right although you can increase this from 3 seconds to 5 in the conftest file. For the main integration test it doesn't need the sleep

Reply:
@dgnball Got the point now, just cleaned up the tests, removed the sleep timer, the new check is now included in the same test, hope this looks okay.